### DISCUSSION TITLE
CycloneDx 1.4|1.5 release

### DISCUSSION THREAD

**Discussion by carlosthe19916** [CycloneDX/cyclonedx-rust-cargo#614] [General]
Hi there. I've read here https://github.com/CycloneDX/cyclonedx-rust-cargo/pull/575 that there is a release planned after supporting CycloneDx 1.4 & 1.5 . Is the release happening soon? 

My apologies if this is not the right place to ask questions.

**Comment by lfrancke** [comment]
Hi!
It's absolutely the right place. Thank you for the question.

We've put off on releasing 1.4 support because there's always _one more improvement_ we need to get to but we just decided that we'll probably do a release within the next two weeks with what we have and then continue work on the 1.5 and 1.6 support.

**Reply by carlosthe19916** [reply]
Awesome! Thanks for your reply! Looking forward to seeing the release!

**Comment by Shnatsel** [comment]
I have shipped `cyclonedx-bom` 0.5.0 that supports CycloneDX v1.4. Support for v1.5 is not yet included; it should come together in the coming months.
